iina/FFmpegController.m

```diff
@@ -18,20 +18,26 @@
 #import <libavutil/mastering_display_metadata.h>
 #pragma clang diagnostic pop
 
+#import "IINA-Swift.h"
+
+#define ERROR(msg, ...) [FFmpegLogger error:([NSString stringWithFormat:(msg), ##__VA_ARGS__])];
+#define LOG(msg, ...) [FFmpegLogger debug:([NSString stringWithFormat:(msg), ##__VA_ARGS__])];
```

Comment:
I would call this INFO or similar personally, "LOG" isn't really a log level

Reply:
The problem was the correct name would be `DEBUG` and that name is taken. I added a `LOG_` prefix to these to allow uniform naming.